code/game/gamemodes/cult/items/tome.dm

```diff
@@ -8,6 +8,70 @@
 	w_class = 2.0
 	unique = TRUE
 	slot_flags = SLOT_BELT
+	var/obj/item/inserted_item  //chalk code graciously stolen from the PDA's pen code   -MalMalmulam
+	var/list/contained_item = list(/obj/item/pen/drafting/) 
```

Comment:
why's this a list, again? this seems like something that should just be a single object

Reply:
Fixed!